crypto/evp/evp_ctx.c

```diff
@@ -220,7 +220,11 @@ int EVP_PKEY_sign_init(EVP_PKEY_CTX *ctx) {
   }
 
   ctx->operation = EVP_PKEY_OP_SIGN;
-  return 1;
+  if ((ctx->pmeth->sign_init == NULL) || (ctx->pmeth->sign_init(ctx))) {
```

Comment:
Do we have any public key algorithm implementations that defines a `sign_init` or `verify_init` function?

Reply:
No, this is imported from OpenSSL. Before this import, no pmeth has sign_init/verify_init.

Reply:
My bad, missed the use in the PSS method.